Client/src/main/resources/fabric.mod.json

```diff
@@ -0,0 +1,33 @@
+{
+	"schemaVersion": 1,
+	"id": "tasbattle",
+	"version": "1.0.0",
+	"name": "TASBattle",
+	"description": "Client for slow motion PVP",
+	"authors": [
+		"Pancake",
+		"Scribble"
+	],
+	"contact": {
+		"homepage": "https://minecrafttas.com",
+		"sources": "https://github.com/MinecraftTAS/TASBattle"
+	},
+
+	"license": "GPLv3",
+	"icon": "",
+	"environment": "client",
+	"accessWidener": "tasbattle.accesswidener",
+	"entrypoints": {
+		"main": [
+			"com.minecrafttas.tasbattle.TASBattle"
+		]
+	},
+	"mixins": [
+		"tasbattle.mixin.json"
+	],
+	"depends": {
+		"fabricloader": ">=0.14.19",
+		"minecraft": "~1.19.4",
+		"java": ">=17"
```

Comment:
and this is why it doesn't launch

Reply:
Nope, this is from the fabric example mod and they should probably be correct https://github.com/FabricMC/fabric-example-mod/blob/38f93b4e0744a819272bf7077bfeed76cb3e7d14/src/main/resources/fabric.mod.json#L29